src/main/java/pl/pszczolkowski/mustdo/domain/task/exception/TasksListAlreadyExistException.java

```diff
@@ -0,0 +1,11 @@
+package pl.pszczolkowski.mustdo.domain.task.exception;
+
+public class TasksListAlreadyExistException
+   extends RuntimeException {
+
+   private static final long serialVersionUID = -1794537307362685773L;
+
+   public TasksListAlreadyExistException() {
+      super("Task list with such name already exists");
```

Comment:
Maybe some additional information that list with that name exists but in the board would be better